src/mutations.js

```diff
@@ -0,0 +1,22 @@
+define([], function() {
+
+  function determineMutationObserver(window) {
```

Comment:
@rrees Server side rendering still does not work.
You pass the `window` to the function. And `window` is not defined on server.
I guess you added this parameter only for testing purposes. But it nullifies the intent of this patch.
May be you can mock window object inside the test? So we do not need to pass it as a parameter.